config/config.default.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const path = require('path');
+
+module.exports = function(appInfo) {
+  const exports = {};
+
+  /**
+   * View options
+   * @member Config#view
+   * @property {String} loadpath - full path of template, default to `app/views`, support multi with comma
```

Comment:
full path of templates, default to `app/views`, multiple paths are supported when paths separated by commas.